config/locales/client.en.yml

```diff
@@ -3280,6 +3280,7 @@ en:
           one: "Sorry, you can only upload %{count} file at a time."
           other: "Sorry, you can only upload %{count} files at a time."
         upload_not_authorized: "Sorry, the file you are trying to upload is not authorized (authorized extensions: %{authorized_extensions})."
+        no_uploads_authorized: "Sorry, no files are allowed to be uploaded."
```

Comment:
Looking at this description, I wonder if this error is even useful. Should we just disable the upload button completely if there are not authorized extensions? Kind of odd that we're telling the user that no files are allowed to be uploaded after they have uploaded.